airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/partition_reader.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from queue import Queue
+
+from airbyte_cdk.sources.streams.concurrent.partitions.partition import Partition
+from airbyte_cdk.sources.streams.concurrent.partitions.types import PartitionCompleteSentinel, QueueItem
+
+
+class PartitionReader:
+    """
+    Generates records from a partition and puts them in a queuea.
+    """
+
+    def __init__(self, queue: Queue[QueueItem]) -> None:
+        """
+        :param queue: The queue to put the records in.
+        """
+        self._output_queue = queue
```

Comment:
Should this be called `queue` instead of `output_queue` so we don't imply it's a separate queue from the PartitionEnqueuer?

Reply:
Sure, done!